examples/browser-webrtc/src/lib.rs

```diff
@@ -59,7 +55,6 @@ pub async fn run(libp2p_endpoint: String) -> Result<(), JsError> {
 #[derive(NetworkBehaviour)]
 struct Behaviour {
```

Comment:
We no longer need this struct now, you can straight reference the `ping::Behaviour` in the builder.